third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1110,8 +1110,7 @@ function createPauseObject(rrpId, level) {
   const className = isCdpObjectProxy(cdpObj) ? "Proxy" : (cdpObj.className || "Function");
 
   // NOTE: `persistentId` is added in V8 → `injected-script.cc`
-  const { persistentId } = cdpObj;
-
+  let { persistentId } = cdpObj;
```

Comment:
What is this change for?  Why "let" instead of "const"?

Reply:
Baggage from a reverted change